### PR TITLE
need to be able to specify the intended minimumAvailability for the t…

### PR DESCRIPTION
**Pull request by geogolem**
…itles added from this list

#### Database Migration
NO

#### Description

this was missing from develop so I added it back in.
#### Todos
- [ ] Tests

#### Issues Fixed or Closed by this PR

* #

### CODE REVIEW

**Comment by geogolem** [comment]
> I think the standard practice is for curly braces to be on a new line

I dont know what happened there.. but it should be fixed now.

**Comment by fryfrog** [comment]
This'll be nice to have again! :)

File: src/Radarr.Api.V2/NetImport/NetImportResource.cs [hunk 1]
```diff
@@ -1,9 +1,9 @@
 using NzbDrone.Core.Movies;
 using NzbDrone.Core.NetImport;
 
-namespace Radarr.Api.V2.NetImport
+namespace Radarr.Api.V2.NetImport
```
Comment (onedr0p) [inline src/Radarr.Api.V2/NetImport/NetImportResource.cs]: Remove whitespace

File: src/Radarr.Api.V2/NetImport/NetImportResource.cs [hunk 2]
```diff
@@ -1,9 +1,9 @@
 using NzbDrone.Core.Movies;
 using NzbDrone.Core.NetImport;
 
-namespace Radarr.Api.V2.NetImport
+namespace Radarr.Api.V2.NetImport 
 {
-    public class NetImportResource : ProviderResource
+    public class NetImportResource : ProviderResource
```
Comment (onedr0p) [inline src/Radarr.Api.V2/NetImport/NetImportResource.cs]: Remove whitespace

File: src/Radarr.Api.V2/NetImport/NetImportResource.cs [hunk 3]
```diff
@@ -15,38 +15,40 @@ public class NetImportResource : ProviderResource
 
     public class NetImportResourceMapper : ProviderResourceMapper<NetImportResource, NetImportDefinition>
     {
-        public override NetImportResource ToResource(NetImportDefinition definition)
+        public override NetImportResource ToResource (NetImportDefinition definition)
```
Comment (onedr0p) [inline src/Radarr.Api.V2/NetImport/NetImportResource.cs]: Remove whitespace

File: src/Radarr.Api.V2/NetImport/NetImportResource.cs [hunk 4]
```diff
@@ -15,38 +15,40 @@ public class NetImportResource : ProviderResource
...
         {
-            if (definition == null)
+            if (definition == null)
```
Comment (onedr0p) [inline src/Radarr.Api.V2/NetImport/NetImportResource.cs]: Remove whitespace

File: src/Radarr.Api.V2/NetImport/NetImportResource.cs [hunk 5]
```diff
@@ -15,38 +15,40 @@ public class NetImportResource : ProviderResource
...
+        public override NetImportResource ToResource (NetImportDefinition definition)
         {
-            if (definition == null)
+            if (definition == null) 
             {
                 return null;
             }
 
-            var resource = base.ToResource(definition);
+            var resource = base.ToResource (definition);
```
Comment (onedr0p) [inline src/Radarr.Api.V2/NetImport/NetImportResource.cs]: Remove whitespace

File: src/Radarr.Api.V2/NetImport/NetImportResource.cs [hunk 6]
```diff
@@ -15,38 +15,40 @@ public class NetImportResource : ProviderResource
...
             resource.ShouldMonitor = definition.ShouldMonitor;
             resource.RootFolderPath = definition.RootFolderPath;
             resource.QualityProfileId = definition.ProfileId;
+            resource.MinimumAvailability = definition.MinimumAvailability;
 
             return resource;
         }
 
-        public override NetImportDefinition ToModel(NetImportResource resource)
+        public override NetImportDefinition ToModel (NetImportResource resource)
```
Comment (onedr0p) [inline src/Radarr.Api.V2/NetImport/NetImportResource.cs]: Remove whitespace

File: src/Radarr.Api.V2/NetImport/NetImportResource.cs [hunk 7]
```diff
@@ -15,38 +15,40 @@ public class NetImportResource : ProviderResource
...
         {
             if (resource == null)
             {
                 return null;
             }
 
-            var definition = base.ToModel(resource);
+            var definition = base.ToModel (resource);
```
Comment (onedr0p) [inline src/Radarr.Api.V2/NetImport/NetImportResource.cs]: Remove whitespace

File: frontend/src/Settings/NetImport/NetImport/EditNetImportModalContent.js [hunk 8]
```diff
@@ -55,128 +57,139 @@ function EditNetImportModalContent(props) {
       <ModalBody>
         {
           isFetching &&
-            <LoadingIndicator />
+          <LoadingIndicator />
```
Comment (Qstick) [inline frontend/src/Settings/NetImport/NetImport/EditNetImportModalContent.js]: Standard form in the code base is to indent after conditional, this should be left alone

File: frontend/src/Settings/NetImport/NetImport/EditNetImportModalContent.js [hunk 9]
```diff
@@ -55,128 +57,139 @@ function EditNetImportModalContent(props) {
...
         }
 
         {
           !isFetching && !!error &&
-            <div>Unable to add a new list, please try again.</div>
+          <div>Unable to add a new list, please try again.</div>
```
Comment (Qstick) [inline frontend/src/Settings/NetImport/NetImport/EditNetImportModalContent.js]: Same as above, leave indented

File: frontend/src/Settings/NetImport/NetImport/EditNetImportModalContent.js [hunk 10]
```diff
@@ -55,128 +57,139 @@ function EditNetImportModalContent(props) {
...
-                      providerData={item}
-                      {...field}
-                      onChange={onFieldChange}
-                    />
-                  );
-                })
-              }
-
-            </Form>
+          <Form
```
Comment (Qstick) [inline frontend/src/Settings/NetImport/NetImport/EditNetImportModalContent.js]: Same as above

File: frontend/src/Settings/NetImport/NetImport/EditNetImportModalContent.js [hunk 11]
```diff
@@ -55,128 +57,139 @@ function EditNetImportModalContent(props) {
...
         {
           id &&
-            <Button
-              className={styles.deleteButton}
-              kind={kinds.DANGER}
-              onPress={onDeleteNetImportPress}
-            >
-              Delete
-            </Button>
+          <Button
```
Comment (Qstick) [inline frontend/src/Settings/NetImport/NetImport/EditNetImportModalContent.js]: Same as above

**Comment by Qstick** [comment]
In addition to above comments, please fix the PR title to match standards and squash your commits down.

File: frontend/src/Settings/NetImport/NetImport/EditNetImportModalContent.js (hunk 8 above)
Comment (geogolem) [reply frontend/src/Settings/NetImport/NetImport/EditNetImportModalContent.js]: I tried to make these changes, but got a ton of lint errors. so I just loaded it up in VS Code and ddid SHIFT-ALT F and it reverted the changes.

What is the best way I can configure vs code to lint automatically? With my last PR I had issues but linting with vs code seemed to resolve them. In this case, it seems VS code is not doing what we want?

Comment (onedr0p) [reply frontend/src/Settings/NetImport/NetImport/EditNetImportModalContent.js]: I usually don't let VSCode format the whole file and try to follow precisely what's already written. Maybe there is a way to auto-format and have the linter be okay with it. I dunno.

**Pull request closed without merging**